routes/socket/report.js

```diff
@@ -7,6 +7,15 @@ const AEM_ALTS = (module.exports.AEM_ALTS = [
 	'BigbyWolf',
 	'Picangel',
 	'birdy',
+	'moira', 
+	'SilentPlayer', 
+	'SilentSheep', 
+	'casdude', 
+	'Naizea', 
+	'Joz', 
+	'Leftwinger', 
+	'hitler4real', 
+	'hitler2real',
```

Comment:
TRIAL_MODS should be directly appended to the staffUserNames list further down, instead of putting trials into two locations.